packages/tasks/src/bitcoinDepositAndCall.ts

```diff
@@ -100,12 +77,11 @@ const makeTransaction = async (
 
 const main = async (args: any, hre: HardhatRuntimeEnvironment) => {
   const TESTNET = bitcoin.networks.testnet;
-  const API = getEndpoints("blockcypher", "btc_testnet")[0].url;
 
-  const pk = process.env.PRIVATE_KEY as any;
+  const pk = args.privateKey || (process.env.BTC_PRIVATE_KEY as any);
```

Comment:
Would this mean that users can provida a pk as an argument on the cli command? That sounds dangerous, the command will still be lingering in their bash history.

Reply:
For testing purposes, it can be convenient especially when new accounts are being generated on localnet in CI, for example.